test/Unit/Api/Resources/SkuTest.php

```diff
@@ -25,24 +25,4 @@ public function testUpdatePassesThroughToConnection()
 
         $sku->update();
     }
-
-    public function testOptionsPassesThroughToConnection()
```

Comment:
options are not passed through to connection anymore, so this expectation does not exist